core/file_api.php

```diff
@@ -761,7 +761,7 @@ function file_clean_name( $p_filename ) {
  */
 function file_generate_unique_name( $p_filepath ) {
 	do {
-		$t_string = md5( crypto_generate_random_string( 32, false ) );
+		$t_string = md5( random_bytes( 32 ) );
```

Comment:
I'd suggest `bin2hex(random_bytes(16))` instead, but really I doubt this matters :)

Reply:
I think there is minor benefit to having the newly generated filenames be consistent to existing ones in terms of length and format.